python/opae.admin/opae/admin/tools/pci_device.py

```diff
@@ -116,6 +118,18 @@ def __call__(self, device, args, *rest):
         action = myargs.action or 'dump'
         getattr(self, action)(device, *rest)
 
+    def clear(self, device):
+        """
+        clear aer errors and print error status.
+        """
+        try:
+            call_process("setpci -s " + str(device) +
+                         " ECAP_AER+0x10.L=0xFFFFFFFF")
+            output = call_process("setpci -s " + str(device) +
+                                  " ECAP_AER+0x10.L")
+            print("aer clear errors:", output)
```

Comment:
why are we printing them out again?

Reply:
checking output( ECAP_AER+0x10.L) after clearing errors